getInfo.py

```diff
@@ -60,12 +76,30 @@ def get_earnings_history(company_ticker: str, context: Optional[ssl.SSLContext]
 def date_time_since_ref(date_object: datetime, reference_date: datetime) -> int:
     """
     Returns the number of days between the given date and the reference date.
+
+    Args:
+        date_object (datetime): Date to calculate the difference from.
+        reference_date (datetime): Reference date to calculate the difference to.
+    
+    Returns:
+        date_object - reference_date (int): Number of days between the two dates.
     """
     # Calculate the number of days between the date and the reference date
     return date_object - reference_date
 
 
 def earnings_since_time(dates: List, start_date: str) -> List:
+    """
+    This function will return a list of earnings since the list of dates
+    and the reference date.
+
+    Args:
+        dates (List): List of dates to calculate the difference from.
+        start_date (str): Reference date to calculate the difference to.
+    
+    Returns:
+        List: List of earnings since the reference date using date_time_since_ref(date, reference_date)
```

Comment:
Is this supposed to be List or list, because I see you use both. I would prefer if these were standardized.